src/resume/resumeStyle.css

```diff
@@ -75,33 +78,41 @@ header {
     height: 30vh;
     min-width: 250px;
     text-align: center;
+    border-bottom: 1px solid gray;
 }
 
 header h3{
     font-size: 34px;
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
 }
 
 #experience {
-    background-color: whitesmoke;
-    height: 30vh;
+    background-color: white;
+    height: 65vh;
     width: 37.5vw;
     float: left;
```

Comment:
use of the `vh` and `vw` units + `float` to get these two sections side by side is very clever! great!